### DISCUSSION TITLE
小界面模式长字段 使用不换行特性会 把label空间挤掉

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#2999] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/8428709/71729748-cc4d-43b2-b485-965218cd9fb9)

**Comment by bb-auto** [comment]
@densen2014 Thank you for filing this issue. In order for us to investigate this issue, please provide [a minimalistic repro project](https://github.com/dotnetcore/bootstrapblazor/blob/main/docs/repro.md) that illustrates the problem.

**Comment by ArgoZhang** [comment, marked as answer]
@densen2014 应该设置了不换行导致的

<img width="332" alt="image" src="https://github.com/dotnetcore/BootstrapBlazor/assets/22001478/6d803477-5369-4fdc-98c8-a231bd9e36a9">
